tests/test_columns.py

```diff
@@ -75,19 +75,19 @@ def test_select_column_using_expression():
 
 def test_select_column_using_case_when():
     sql = """INSERT OVERWRITE TABLE tab1
-SELECT CASE WHEN col1 = 1 THEN "V1" WHEN col1 = 2 THEN "V2" END
+SELECT CASE WHEN col1 = 1 THEN 'V1' WHEN col1 = 2 THEN 'V2' END
```

Comment:
Note this change. Is it OK or is there a reason to consider double quoted strings as string literals here? This behavior is not consistent between RDBMS's, but being used to Postgres, to me "V1" and "V2" would intuitively correspond to column names instead of literals